### DISCUSSION TITLE
Не работает instagram

### DISCUSSION THREAD

**Discussion by EvgenyKV** [hufrea/byedpi#57] [General]
Пишет, что не удается найти сайт, facebook и многие другие сайты при этом работают, пробовал
--disorder 1 --auto=torst --tlsrec 1+s
и
--fake -1 --ttl 8
DNS работает через DoH
в чем дело?

**Comment by zimbabwe303** [comment]
Инстаграм (как и Твиттер) забанены хардкорно, по айпишникам. Тут поможет только VPN. У них, конечно, как у любого междупланетного сайта, балансировщик нагрузки и зеркала, поэтому айпишников там много, и из них один или два доступны через byedpi, поэтому если перезагружать страницу снова и снова, то она откроется, но там всё равно далеко не уйдёшь - балансировщик снова переключит айпишник, и опять в бан.

**Reply by EvgenyKV** [reply]
Твитор пашет, но без видео и картинок

**Reply by zimbabwe303** [reply]
> Твитор пашет, но без видео и картинок

Далеко не у всех. Если в HOSTS добавить пока ещё незабаненные айпишники (на ntc.party вроде люди делились), то пашет, а так нет. Всё очень неоднородно по стране, почему-то. Хотя вроде как везде одни и те же ТСПУ установили. Вообще много странностей в поведении всей этой фигни.
